Sources/Quick/QuickTestSuite.swift

```diff
@@ -36,6 +36,7 @@ public class QuickTestSuite: XCTestSuite {
      It is expected that the first call should return a valid test suite, and
      all subsequent calls should return `nil`.
      */
+    @objc
```

Comment:
The file is entirely surrounded with `#if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)`.